helper/resource/testing.go

```diff
@@ -191,15 +191,18 @@ func filterSweepers(f string, source map[string]*Sweeper) map[string]*Sweeper {
 // Since filterSweepers performs fuzzy matching, this function is used
 // to perform exact sweeper and dependency lookup.
 func filterSweeperWithDependencies(name string, source map[string]*Sweeper) map[string]*Sweeper {
-	currentSweeper := source[name]
 	result := make(map[string]*Sweeper)
 
-	result[name] = currentSweeper
+	if currentSweeper, ok := source[name]; ok {
+		result[name] = currentSweeper
```

Comment:
Do you mind reducing the nesting levels (and improve readability) here by "exiting early"? i.e.

```go
currentSweeper, ok := source[name]
if !ok {
    log.Printf("[DEBUG] Sweeper has dependency (%s), but that sweeper was not found", name)
    return result
}
```

Reply:
Done.